---
title: "Abiotic and Biotic Characteristics of Crystal Bog, Wisconsin in 2007 and 2009"
author: "Megan Worth"
date: "12/14/2021"
output: pdf_document
editor_options: 
  chunk_output_type: console
---
**Introduction**
  
  Microbial communities are responsible for carrying out crucial ecosystem functions and processes, and are therefore essential to life on Earth (Li et al., 2015). Specifically, in aquatic systems, bacterial communities play vital roles in biogeochemical cycling (Monard et al., 2016). Knowledge of the composition, structure, and distribution of microbial communities can be utilized to predict how an ecosystem may respond to environmental changes over time (Monard et al., 2016; Sieber et al., 2020).
  The North Temperate Lakes Long Term Ecological Research (LTER) program located in Vilas County, Wisconsin has collected a myriad of data from lakes and bogs in the area. In 2007 and 2009, data on pH, dissolved oxygen (DO), and temperature were collected from Crystal Bog. Additionally, samples were collected and analyzed to identify members of the microbial community by 16S rRNA sequencing (McMahon et al., 2019). This dataset was analyzed using R v. 4.1.1 to gain an understanding of the influence these environmental factors (pH, DO, temperature) may have had on microbial community composition between Summer 2007 and Summer 2009.

**Methods**

  Data was collected by researchers through the North Temperate Lakes LTER in Vilas County, Wisconsin. Freshwater samples were collected from Crystal Bog in 2007 and 2009 and 16S rRNA sequencing was performed. Data was analyzed using R v. 4.1.1. 
  
**Results and Discussion**
  Abiotic characteristics of Crystal Bog measured in Summer 2007 and Summer 2009 were temperature (C) and DO content (mg/L). Measurements of pH were recorded only in 2007. Between June and August of 2007 and 2009, the temperature of Crystal Bog increased consistently (Figure 1A). In 2007, the DO content increased between June and August, while it decreased over the same time span in 2009 (Figure 1B). Average DO content plotted against depth (m) showed a decreasing amount of oxygen as depth increased (Figure 2). The pH of Crystal Bog in 2007 became more acidic between June and July, and increased between July and August (Figure 1C). The average pH of Crystal Bog remained acidic, with no measurements recorded over a pH of 4.0. 
  16S rRNA sequencing of samples resulted in the recognition of 45 unique phyla across all samples in 2007 and 2009. The total abundance of each phylum ranged from 20 (FCPU426) to 5,597,312 (Proteobacteria) reads (Table 1). Samples were collected at varying depths, including both the upper (Epilimnion) layer and lower (Hypolimnion) layer of Crystal Bog. When compared, the same five major phyla (Acidobacteria, Actinobacteria, Bacteroidetes, Proteobacteria, and Verrucomicrobia) were found to dominate both layers (Figure 3). Relative abundances of the major phyla (>200 reads) were calculated separately for 2007 and 2009 due to varying numbers of samples collected per year. In 2007, eight phyla were found to make up the majority of the microbial community. In order of abundance, the eight phyla were: Proteobacteria, Actinobacteria, Acidobacteria, Verrucomicrobia, Bacteroidetes, Fusobacteria, Cyanobacteria, and OD1 (Figure 4A). In 2009, only five phyla were found to dominate the microbial community. In order of abundance, the five phyla were: Proteobacteria, Actinobacteria, Acidobacteria, Verrucomicrobia, and Bacteroidetes (Figure 4B).
  


```{r setup, include=FALSE, message = FALSE}
knitr::opts_chunk$set(echo = TRUE)
```

```{r, echo = FALSE, message = FALSE, warning = FALSE}

sup_data <- read.csv("C:/Users/megan/Desktop/806_final/supp_data1.csv", stringsAsFactors = TRUE)

lake_data <- read.csv("C:/Users/megan/Desktop/806_final/lake_data1.csv", stringsAsFactors = TRUE)

otu_data <- read.csv("C:/Users/megan/Desktop/806_final/otu_data1.csv", stringsAsFactors = TRUE)

rel_abun <- read.csv("C:/Users/megan/Desktop/806_final/rel_abun1.csv", stringsAsFactors = TRUE)

```

```{r, echo = FALSE, message = FALSE, warning = FALSE}
library(dplyr)
library(tidyr)
library(ggplot2)
```

```{r, echo = FALSE, message = FALSE, warning = FALSE}

comb1 <- inner_join(otu_data, rel_abun, by = "OTU")
comb2 <- distinct(inner_join(sup_data, lake_data))
database <- distinct(inner_join(comb1, comb2))


```

```{r, echo = FALSE, message = FALSE, warning = FALSE}

select(database, Month, Year, Temperature) %>%
  group_by(Year, Month) %>%
  filter(Temperature != "NA", Month == 6 | Month ==
  7 | Month == 8) %>%
  summarize(mean_Temperature = mean(Temperature)) %>%
  ggplot(mapping = aes(x = Month, y = mean_Temperature, 
  color = as.factor(Year),
  group = Year, by = Year)) + geom_point() + geom_line() +
  labs(color = "Year", x = "Month", y = "Temperature (C)") +
  scale_color_brewer(palette = "Dark2") + scale_x_continuous(
  breaks=c(6, 7, 8)) -> temp_graph

```

```{r, echo = FALSE, message = FALSE, warning = FALSE}

select(database, Month, Year, DO) %>%
  group_by(Year, Month) %>%
  filter(DO != "NA", Month == 6 | Month == 7 | Month == 8) %>%
  summarize(mean_DO = mean(DO)) %>%
  ggplot(mapping = aes(x = Month, y = mean_DO, color =
  as.factor(Year), group = Year, by = Year)) + geom_point() + 
  geom_line() + labs(color = "Year", x = "Month", 
  y = "Dissolved Oxygen (mg/L)") + scale_color_brewer(palette = "Dark2") +
  scale_x_continuous(breaks=c(6, 7, 8)) -> DO_graph

```

```{r, echo = FALSE, message = FALSE, warning = FALSE}

select(database, Month, Year, pH) %>%
  group_by(Year, Month) %>%
  filter(pH != "NA", Month == 6 | Month == 7 | Month == 8) %>%
  summarize(mean_pH = mean(pH)) %>%
  ggplot(mapping = aes(x = Month, y = mean_pH, color =
  as.factor(Year), group = Year)) + geom_point() +
  geom_line() + labs(color = "Year", x = "Month", y = "pH") +
  scale_color_brewer(palette = "Dark2") + scale_x_continuous(
  breaks = c(6, 7, 8)) -> pH_graph

```

```{r, echo = FALSE, message = FALSE, warning = FALSE, fig.cap= "Average monthly temperature (A), dissolved oxygen (DO) content (B), and pH (C) of Crystal Bog in 2007 and 2009. Time scale covers the Summer months: June (6), July (7), and August (8)."}

library(ggpubr)
ggarrange(temp_graph, DO_graph, pH_graph, labels = c("A", "B", "C"),
  common.legend = TRUE, legend = "bottom", ncol = 2, nrow = 2)

```

```{r, echo = FALSE, message = FALSE, warning = FALSE, fig.cap = "Average dissolved oxygen (DO) content (mg/L) measured at increasing depth (m) in Crystal Bog."}

select(database, Depth, DO, Month, Year) %>%
  filter(DO != "NA", Depth != "NA", Month == 6 | Month ==
  7 | Month == 8, Year == "2009") %>%
  group_by(Depth) %>%
  summarize(mean_DO = mean(DO)) %>%
  ggplot(mapping = aes(x = Depth, y = mean_DO)) +
  geom_point() + geom_line() + labs(x = "Depth (m)", y =
  "DO (mg/L)")

```

```{r, echo = FALSE, message = FALSE, warning = FALSE, }
library(knitr)
select(database, Phylum, value) %>%
  group_by(Phylum) %>%
  filter(value > 0) %>%
  summarize(value = sum(value)) %>%
  kable(., caption = "Total abundances of each phyla in Crystal Bog", col.names = c("Phylum", "Total Reads"))   
```

```{r, echo = FALSE, message = FALSE, warning = FALSE, fig.cap= "Relative abundances of the most abundant phyla (> 200 reads) identified at the upper (Epilimnion) and lower (Hypolimnion) layers of Crystal Bog."}

select(database, Phylum, value, Layer, Year) %>%
  filter(Year == 2009, value > 200) %>%
  group_by(Layer, Phylum) %>%
  summarize(value=sum(value)) %>%
  ggplot(mapping = aes(fill = Phylum, x =
  Layer, y = value, color = Phylum)) + 
  geom_bar(position = "stack", stat="identity") +
  geom_text(aes(label = Phylum), size = 3, color =
  "black", position = position_stack(vjust = 0.5)) +
  labs(x = "Layer", y = "Abundance") +
  theme(legend.text = element_text(size = 6), 
  legend.key.size = unit(0.35, "cm"), 
  legend.key.height = unit(0.35, "cm"),  
  legend.key.width = unit(0.35, "cm")) 

```

```{r, echo = FALSE, message = FALSE, warning = FALSE}

select(database, Phylum, value, Year, Month) %>%
  filter(value > 200, Year == 2007, Month ==
  6 | Month == 7 | Month == 8) %>%
  group_by(Year, Phylum) %>%
  summarize(value=sum(value)) %>%
  ggplot(mapping = aes(x = Phylum, y = 
  value)) + geom_bar(stat = "identity", fill =
  "steelblue") + ggtitle("2007") + labs(x = "Phylum",
  y = "Abundance") + theme(axis.text.x = element_text(
  angle = 90, vjust = 0.5)) -> graph_1

```

```{r, echo = FALSE, message = FALSE, warning = FALSE}

select(database, Phylum, value, Year, Month) %>%
  filter(value > 200, Year == 2009, Month ==
  6 | Month == 7 | Month == 8) %>%
  group_by(Year, Phylum) %>%
  summarize(value=sum(value)) %>%
  ggplot(mapping = aes(x = Phylum, y = 
  value)) + geom_bar(stat = "identity", fill =
  "Coral2") + ggtitle("2009") + labs(x =
  "Phylum", y = "Abundance") + theme(axis.text.x =
  element_text(angle = 90, vjust = 0.5))-> graph_2
```

```{r echo = FALSE, message = FALSE, warning = FALSE, fig.cap = "Relative abundances of the dominant phyla (>200 reads) in 2007 (A) and 2009 (B)."}

ggarrange(graph_1, graph_2, labels = c("A", "B"),
  ncol = 2, nrow = 1)

```

**References**

Li P, Yang SF, Lv BB, Zhao K, Lin MF, Zhou S, Song X, Tang XM. 2015. Comparison of extraction methods of total microbial DNA from freshwater. Genet Mol Res. 14(1):730-8.

Monard, C., Gantner, S., Bertilsson, S., Hallin, S., & Stenlid, J. 2016. Habitat generalists and specialists in microbial communities across a terrestrial-freshwater gradient. Scientific reports, 6, 37719.

Sieber, G., Beisser, D., Bock, C., & Boenigk, J. 2020. Protistan and fungal diversity in soils and freshwater lakes are substantially different. Scientific reports, 10(1), 20025.

McMahon, K., S. Jones, A. Shade, R. Newton, E. Read, L. Beversdorf, R. Rohwer, B. Peterson, A. Linz, E. McDaniel, 	G. 	Wolf, and S. Schmitz. 2019. Microbial Observatory at North Temperate Lakes LTER High-resolution temporal and 	spatial dynamics of microbial community structure in freshwater bog lakes 2005 - 2009 original format ver 4. 	Environmental Data Initiative.
